### DISCUSSION TITLE
Ayuda como subir las practicas

### DISCUSSION THREAD

**Discussion by AdoRockdriguez** [Launch-X-Latam/MisionFrontEnd#484] [Team Posts]
Hola!
Por favor me pueden ayudar a saber donde se suben o como se postean las practicas, voy atrasado en los lives y estoy iniciando CSS apenas. Gracias

**Comment by gfrodriguez** [comment]
[247](https://github.com/Launch-X-Latam/MisionFrontEnd/discussions/247)

**Comment by DiegoIrribarren** [comment]
Alguien sabe, que hay. Hacer en la práctica uno? Me he liado.. No entiendo
exactamente que hay que hacer


Diego V. Irribarren

**Comment by Ed-GoFu** [comment]
Hola! No se si ya obtuviste respuesta pero me parece que para mañana hay que entregar la práctica que se muestra en la sig. página. https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/02%20-%20HTML/practicas
Saludos! Si alguien pudiera confirmar esta información, porque yo también tengo duda si existe algo más que necesitemos entregar.

Edgar González

**Comment by JoseEduardoVelazquezCN** [comment]
Me parece que aún no hay fecha definida para la entrega, en su defecto mañana hay live con el Sherpa y se tienen que entregar ahí seguro nos informa en dónde subiremos nuestras prácticas
Pero por lo que he visto las fechas están desactualizadas y deben ser de otra edición del programa launch X

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by VictorRuelas** [comment]
Que tal compañeros este grupo es el de Backend, por ahí en la organización está el de Frontend. Solo como comentario.

Saludos.

**Comment by FaustoPaFl** [comment]
El MC dijo en la última sesión que no hay un repositorio o lugar específico para subirlas, en realidad son para que las tengas en tu repo como parte de prácticas personales, que te pueden servir para usar como portafolio, es lo que él dijo, suerte.
